### DISCUSSION TITLE
为特定网站指定特定AI专家

### DISCUSSION THREAD

**Discussion by inertia42** [immersive-translate/immersive-translate#1846] [Ideas]
### 新功能 | New Feature

1. 希望能够为特定网站指定特定的AI专家，实现针对不同网站特化 Prompt。例如设置对 reddit 自动使用 reddit 翻译专用 Prompt。

### 为什么您想要添加这些功能 | Why you want to add these features

1. 目前选择 AI 专家后，会使得所有页面上默认的 AI 专家选项被修改，切换到其他网页后需要手动切换回默认或者其他选项，使用起来不太方便。

**Comment by keyboard3** [comment]
![image](https://github.com/immersive-translate/immersive-translate/assets/13177744/0fec35b5-6fe1-4a93-ac0a-2bdffc636c32)

![image](https://github.com/immersive-translate/immersive-translate/assets/13177744/d7ffe432-0143-4e4d-a07f-f31af153ede3)
选择智能选择就可以了
另外专家也可以指定网址
![image](https://github.com/immersive-translate/immersive-translate/assets/13177744/3bf186fc-04d2-4c73-b1b9-895f1e7e9424)
